scripts/intfstat

```diff
@@ -305,6 +305,19 @@ def main():
 
     if save_fresh_stats:
         try:
+            # Add the information also to the general file - i.e. without the interface name
+            if interface_name != '' and interface_name in cnstat_fqn_file.split('/')[-1]:
+                gen_index = cnstat_fqn_file.rfind('/')
+                cnstat_fqn_general_file = cnstat_fqn_file[:gen_index] + cnstat_fqn_file[gen_index:].split('-')[0]
+                if os.path.isfile(cnstat_fqn_general_file):
+                    try:
+                        data = pickle.load(open(cnstat_fqn_general_file, 'rb'))
+                        for key, val in cnstat_dict.items():
+                            data[key] = val
+                        pickle.dump(data, open(cnstat_fqn_general_file, 'wb'))
+                    except IOError as e:
+                        sys.exit(e.errno)
+            # Add the information also to interface specific file
             pickle.dump(cnstat_dict, open(cnstat_fqn_file, 'wb'))
         except IOError as e:
             sys.exit(e.errno)
```

Comment:
If we are making the general file upto date, can you please explain the need for the interface specific file? In my opinion one file should be good enough to save stats.

Reply:
There is another feature we need to take into consideration - tag_name. 
If a user wants to clear counters with a certain tag, we need to keep stats in this tag file. 
so I removed the interface name in file which is indeed unneeded, but changed the logic in the script to not brake tag_name logic.